zebra/interface.c

```diff
@@ -2933,7 +2934,27 @@ DEFUN (multicast,
 		if_refresh(ifp);
 	}
 	if_data = ifp->info;
-	if_data->multicast = IF_ZEBRA_MULTICAST_ON;
+	if_data->multicast = IF_ZEBRA_DATA_ON;
+
+	return CMD_SUCCESS;
+}
+
+DEFPY (mpls,
+       mpls_cmd,
+       "[no] mpls",
```

Comment:
Might be preferable to call this `mpls enable`?

If we add other MPLS config knobs, they would be `mpls foobar 1234`, having a plain `mpls` command tends to be confusing especially with the `no mpls` form - since presumably that wouldn't remove `mpls foobar 1234`.